samples/bluetooth/direct_test_mode/Kconfig

```
@@ -54,8 +54,44 @@ config DTM_TRANSPORT_TWOWIRE
 	help
 	  Use the Two Wire transport interface as the DTM transport layer.
 
+config DTM_TRANSPORT_HCI
+	bool "DTM over HCI UART [EXPERIMENTAL]"
+	depends on SERIAL
+	depends on NET_BUF
+	select EXPERIMENTAL
+	help
+	  Use the HCI UART transport interface as the DTM transport layer.
+
 endchoice # DTM_TRANSPORT
 
+if DTM_TRANSPORT_HCI
+
+config DTM_HCI_QUEUE_COUNT
+	int "Count of HCI rx/tx queues"
```

```suggestion
	int "Count of HCI RX/TX queues"
```
Capitalize these in all places.